dev/breeze/src/airflow_breeze/prepare_providers/provider_documentation.py

```diff
@@ -656,7 +656,7 @@ def update_release_notes(
 ) -> tuple[bool, bool]:
     """Updates generated files.
 
-    This includes the readme, changes, and/or setup.cfg/setup.py/manifest.in/provider_info.
+    This includes the readme, changes, and/or pyproject.toml.
```

Comment:
Also same but for provider info

Reply:
Ah yes. Here updating provider documentation modifies provider.yaml files. Fixed.